hazelcast-jet-core/src/main/java/com/hazelcast/jet/Sinks.java

```diff
@@ -58,6 +63,82 @@ private Sinks() {
         return new SinkImpl<>(sinkName, metaSupplier);
     }
 
+    /**
+     * Returns a sink that updates {@code Map.Entry}s it receives into a Hazelcast
+     * {@code IMap} with the specified name and update function. The function
+     * will be wrapped inside an {@link EntryProcessor}.
+     *
+     *  @param mapName name of the map
+     *  @param updateFn update function contains the update logic.
+     */
+    public static <K, V, E extends Map.Entry<K, V>> Sink<E> updateMap(
+            String mapName,
+            DistributedBiFunction<Entry<K, V>, ? super V, ? extends V> updateFn
+    ) {
+        return fromProcessor("updateMap(" + mapName + ')', updateMapP(mapName, updateFn));
```

Comment:
Use `updateMapSink(xxx)` here, this was changed in PR #616 that I just merged

Reply:
Will adopt the naming convention from there.